### DISCUSSION TITLE
Q about VIMP figure

### DISCUSSION THREAD

**Discussion by panyue3000** [kogalur/randomForestSRC#368] [Q&A]
To whom it may concern, I have a quick question about the VIMP generated from rfsrc. The images have a funky appearance where if there are too many variables, they overlap with each other. Is there any way that we can specify and select only top number of variables in the figure from rfsrc? Any good suggestions? Thanks.

**Comment by ishwaran** [comment]
If you really want a customized graph, then I recommend just extracting the VIMP from the forest object and plotting it yourself.  For example, here's how to extract VIMP for a simple regression problem:

```
o <- rfsrc(mpg~.,mtcars,importance="permute")
imp <- o$importance
print(imp)
## simple barplot of importance, but you can design something you might like better
barplot(imp)
```
